storybook/storybook-react/config/preview.jsx

```diff
@@ -17,4 +17,9 @@ export const parameters = {
       state: 'open',
     },
   },
+  options: {
```

Comment:
preview still has a `jsx` extension, can we make it a `tsx` extentsion here as well?